unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs

```diff
@@ -106,8 +109,11 @@ public void ReportScene()
     /// <param name="controller">Controller for the players' list HUD</param>
     public void AddUsersAroundIndicator(UsersAroundListHUDController controller)
     {
+        bool peopleCounterEnabled = kernelConfig.features.enablePeopleCounter;
+        
         view.usersAroundListHudButton.gameObject.SetActive(true);
         controller.SetButtonView(view.usersAroundListHudButton);
+        controller.ToggleUsersCount(peopleCounterEnabled);
     }
```

Comment:
Since (I think) this is called only once it could be simplified
```suggestion
    public void AddUsersAroundIndicator(UsersAroundListHUDController controller)
    {
        view.usersAroundListHudButton.gameObject.SetActive(true);
        controller.SetButtonView(view.usersAroundListHudButton);
        controller.ToggleUsersCount(false);
        KernelConfig.i.EnsureConfigInitialized().Then(kc => controller.ToggleUsersCount(kc.features.enablePeopleCounter));
    }
```

Reply:
Done!